### ISSUE TITLE
New build system Errors - Earl Grey

### ISSUE THREAD

**Issue by badrinathvm** [google/EarlGrey#740]
Showing All Messages
:-1: Multiple commands produce '/Users/bvenkatnarayansetty/Library/Developer/Xcode/DerivedData/Lightning-bqwglibfxszbzcbfqlsqwrftorbm/Build/Products/Debug-iphonesimulator/Evernote.app/EarlGrey.framework':

1) Target 'XXXUITests' has copy command from '/Users/bvenkatnarayansetty/Library/Developer/Xcode/DerivedData/Lightning-bqwglibfxszbzcbfqlsqwrftorbm/Build/Products/Debug-iphonesimulator/EarlGrey.framework' to '/Users/bvenkatnarayansetty/Library/Developer/Xcode/DerivedData/Lightning-bqwglibfxszbzcbfqlsqwrftorbm/Build/Products/Debug-iphonesimulator/XXX.app/EarlGrey.framework'
2) Target 'XXXUIFLETests' has copy command from '/Users/bvenkatnarayansetty/Library/Developer/Xcode/DerivedData/Lightning-bqwglibfxszbzcbfqlsqwrftorbm/Build/Products/Debug-iphonesimulator/EarlGrey.framework' to '/Users/bvenkatnarayansetty/Library/Developer/Xcode/DerivedData/Lightning-bqwglibfxszbzcbfqlsqwrftorbm/Build/Products/Debug-iphonesimulator/XXX.app/EarlGrey.framework'
3) Target 'XXXUIEditorTests' has copy command from '/Users/bvenkatnarayansetty/Library/Developer/Xcode/DerivedData/Lightning-bqwglibfxszbzcbfqlsqwrftorbm/Build/Products/Debug-iphonesimulator/EarlGrey.framework' to 

> '/Users/bvenkatnarayansetty/Library/Developer/Xcode/DerivedData/Lightning-bqwglibfxszbzcbfqlsqwrftorbm/Build/Products/Debug-iphonesimulator/XXX.app/EarlGrey.framework'

**Comment by tirodkar** [comment]
Yeah, we use the Copy Files Build Phase to embed EarlGrey into the app under test. Can you tell me how this is done with the new build system and we could check this?

**Comment by tirodkar** [comment]
Copy Phases are related to your Target's Build Phases and not necessarily to the schemes. Is there any new change for embedding frameworks that has been added to the Build system? I'll have to take a look in a bit.

**Comment by badrinathvm** [comment]
what i think is we just need to add it once, so that respective targets will pick up. I am also invetsigatin where to add it.

**Comment by tirodkar** [comment]
@badrinathvm were there any updates here?
